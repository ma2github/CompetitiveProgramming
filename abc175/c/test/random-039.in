110873786713337 110055728834726 240425046481828
