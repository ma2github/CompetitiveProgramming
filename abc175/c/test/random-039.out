110873786713337
